contracts/src/components/movable.cairo

```diff
@@ -16,3 +19,61 @@ struct ArrivalTime {
     entity_id: u128,
     arrives_at: u64,
 }
+
+#[derive(Component, Copy, Drop, Serde, SerdeLen)]
+struct Travel {
```

Comment:
- feels like Movable and Travel serve the same purpose here. I don't see the point of having this info separated in 2 different components, if they will never be used one without the other.
- departure_time should not be stored because it can be computed from the arrival_time and coords
- storing departure_coord_x, departure_coord_y, last_stop_coord_x, last_stop_coord_y + Position component means you have 3 sets of coordinates to store. I think thiis could work with only 2 sets (as we discussed in discord)

Reply:
If we recompute departure_time, it might be incorrect. e.g if the traveling entity uses a road. Unless we want to store that  a road was used